### ISSUE TITLE
[$15]{MSFT-89}{Submit} Missing first level heading

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2799]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/30095032/submit
2. Inspect Element
3. Observe the first level heading

**Expected Result**
First level heading should be present describing the purpose.

**Actual Result**
First level heading do not describe topic or purpose.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.2 Page Titled
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2089

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096172 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096172 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096172 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096172 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097284 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097284 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097284 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Does not found first level heading. 
https://monosnap.com/file/IwZawGuZkSrhqjs9yby0vaucJE9SR6

**Comment by codeMinter** [comment]
> Does not found first level heading.
> https://monosnap.com/file/IwZawGuZkSrhqjs9yby0vaucJE9SR6

@anonymousjaggu ☝️

**Comment by drasticdpk** [comment]
@veshu @anonymousjaggu , First level heading is fine at same time can you please make h2
instead of h4 in Files section.
![image](https://user-images.githubusercontent.com/25297859/62236397-bf06e800-b3ee-11e9-8fd2-d5f6c22c68d1.png)

**Comment by veshu** [comment]
@drasticdpk create another issue with list of other things to handle separately.

**Comment by veshu** [comment]
Opened another issue to handle the new issue separately #3185

**Comment by veshu** [comment]
@drasticdpk I have passed the QA based on your confirmation above. Thanks!

**Comment by drasticdpk** [comment]
First level heading has not updated yet.
![image](https://user-images.githubusercontent.com/25297859/62474950-bbdf7380-b7c3-11e9-8fe9-360e99f150b6.png)

**Comment by crazyk07** [comment]
@veshu @sushilshinde I'm confused by the conversation, but in Prod, there is still no First level Heading.  Please check out to see what the issue is.  Maybe it hasn't went to prod yet?

**Comment by sushilshinde** [comment]
@crazyk07 @veshu you are right this is miss on prod, will go in next deployment

**Comment by tosha5252** [comment]
Verified changed to <h1>

Win10/Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097284<br/><br/>```This is an automated message for crazyk via Topcoder X```
